modules/sdk-coin-ton/test/unit/index.ts

```diff
@@ -0,0 +1,16 @@
+import 'should';
+
+import { TestBitGo, TestBitGoAPI } from '@bitgo/sdk-test';
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { Ton, Tton } from '../../src';
+
+describe('TON:', function () {
```

Comment:
Based on the convention that we are following for most of other coins, the name of this file should be ton.ts instead of index.ts